src/components/Focus/Focus.tsx

```diff
@@ -13,8 +13,19 @@ export const Focus = memo(function Focus({
   root,
 }: FocusProps) {
   useEffect(() => {
-    if (disabled || !root || root.querySelector('[autofocus]')) return;
-    focusFirstFocusableNode(root, false);
+    if (disabled || !root) {
+      return;
+    }
+
+    let node: HTMLElement | null;
+    if ('current' in root) {
+      node = (root as React.RefObject<HTMLElement>).current;
+    } else {
+      node = root as HTMLElement;
+    }
+
+    if (!node || node.querySelector('[autofocus]')) return;
```

Comment:
Personal preference but I am not a fan of this one liner syntax.

Reply:
Will change it - it will also match the early return above